frontend/src/main.js

```diff
@@ -1,9 +1,10 @@
 import Vue from 'vue'
 import App from './App.vue'
 import router from '@/router'
-import { vuetifyLoader, auth, storeLoader, filterLoading, formBaseComponents, ignoreNativeBindingWarnMessages, i18n, veeValidate, vueMoment, moment } from './plugins'
+import { vuetifyLoader, auth, storeLoader, filterLoading, formBaseComponents, ignoreNativeBindingWarnMessages, i18n, veeValidate } from './plugins'
 import { store } from './plugins/store'
 import { vuetify } from './plugins/vuetify'
+import './plugins/dayjs'
```

Comment:
Could you wrap the contents of this file in a small plugin, like we already do for e.g. auth? We have previously had issues with import statements with side effects, see https://github.com/ecamp/ecamp3/pull/616#discussion_r522342206.